test/test_nn.py

```diff
@@ -3952,10 +3952,9 @@ def test_MaxUnpool2d_output_size(self):
             for w in range(3, 10):
                 if 4 <= h <= 6 and 4 <= w <= 6:
                     size = (h, w)
-                    if h == 5:
-                        size = torch.LongStorage(size)
-                    elif h == 6:
-                        size = torch.LongStorage((1, 1) + size)
+                    if h == 6:
```

Comment:
why we are removing h==5 case?

Reply:
It’s just making a torch.Size into a tensor of the same thing (which isn’t explicitly supported in the docs as an input for max_unpool, so that case isn’t really necessary